components/avatar/avatar.stories.js

```diff
@@ -96,6 +96,9 @@ const defaultImage = require('./person.png');
 
 // Stories
 export const Default = DefaultTemplate.bind({});
+Default.decorators = [() => ({
+  template: `<div class="d-d-flex"><story /></div>`,
```

Comment:
Kinda confused why we need this when the wrapper on the component itself already has d-d-flex hmmm....

Reply:
It's not actually a flex parent, we just need to contain the avatar or else it will take all the available width, which on storybook breaks the avatar

Reply:
alright guess we'll see how it goes